orchestrator/src/main/java/com/yahoo/vespa/orchestrator/OrchestratorImpl.java

```diff
@@ -375,13 +380,50 @@ private void setApplicationStatus(ApplicationId appId, ApplicationInstanceStatus
 
                 // If the clustercontroller throws an error the nodes will be marked as allowed to be down
                 // and be set back up on next resume invocation.
-                setClusterStateInController(context.createSubcontextWithinLock(), application, ClusterControllerNodeState.MAINTENANCE);
+                setClusterStateInController(context.createSubcontextWithinLock(), application, MAINTENANCE);
             }
 
             lock.setApplicationInstanceStatus(status);
         }
     }
 
+    @Override
+    public boolean isQuiescent(ApplicationId id) {
```

Comment:
Specify this is about content clusters.  This may return true even though all jdisc containers are down.